vignettes/queries_dataset.Rmd

```diff
@@ -0,0 +1,178 @@
+---
+title: "Queries dataset documentation"
+output:
+  rmarkdown::html_vignette:
+  toc: false
+vignette: >
+  %\VignetteIndexEntry{Queries dataset documentation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+  ```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+To support the analysis of the safety analysis, it is quite common to define specific grouping of events. One of the most common way is to group events or medications by a specific medical concept such as a Standard MedDRA Queries or WHO-Drug Standardized Drug Grouping . 
```

Comment:
Can this change this to `To support the safety analysis`? It currently doesn't sound very clear. Could also remove the comma.
Change `way` to `ways`.
Remove space before full stop.

Reply:
done thanks